go.mod

```diff
@@ -69,7 +69,7 @@ require (
 	github.com/joho/sqltocsv v0.0.0-20210428211105-a6d6801d59df
 	github.com/kisielk/errcheck v1.6.3
 	github.com/klauspost/compress v1.17.1
-	github.com/ks3sdklib/aws-sdk-go v1.2.9
+	github.com/ks3sdklib/aws-sdk-go v1.2.7
```

Comment:
why? we just upgrade it. https://github.com/pingcap/tidb/pull/50179

Reply:
There is a network problem with ks3 `Read()`. I want to make a quick test by downgrade the version.